javascript/javascript2/week1/spirit.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Spirit Animal Name Generator</title>
+  </head>
+  <body>
+    <label for="nameInput">Enter your name:</label>
+    <input type="text" id="nameInput" />
+    <button id="generateButton">Generate Spirit Animal</button>
+    <p id="spiritAnimalText"></p>
+
+    <label for="eventSelect">Choose event to generate spirit animal:</label>
+    <select id="eventSelect">
+      <option value="1">Click Button</option>
```

Comment:
Prefer using explicit values to make your `switch` in JS more readable
```suggestion
      <option value="click">Click Button</option>
```